### PR TITLE
Add PL figure element example to exampleCourse

### PR DESCRIPTION
**Pull request by bojinyao**
Fixes https://github.com/ace-lab/pl-ucb-cs10/issues/51

Add examples of using various attributes of `<pl-figure>` to exampleCourse/element

### CODE REVIEW

**Comment by bojinyao** [comment]
The "sin-x.png" was generated by a royalty free graph generator. 
The "true.png" "false.png" and "boolean.png" came from Berkeley's Snap (https://snap.berkeley.edu/about) and is publicly licensed under GNU Affero General Public License (https://snap.berkeley.edu/tos)

File: exampleCourse/questions/element/figure/question.html [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+            plt.title(f"{coeff} * cos(x)")
+            plt.xlabel("x values")
+            plt.ylabel("y values")
+            plt.ylim(-6, 6)
+    
+            # save plot as utf-8 bytes
+            f = io.BytesIO()
+            plt.savefig(f, format='png')
+            return f    
+</pl-code>
```
[the remark this answers is not in the document]
Comment (bojinyao) [reply exampleCourse/questions/element/figure/question.html]: Resolved 👍

File: exampleCourse/questions/element/figure/question.html [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        <pl-multiple-choice answers-name="booleans">
+            <pl-answer correct="true">Select this: <pl-figure file-name="true.png" inline="true"></pl-figure></pl-answer>
+            <pl-answer>Not this: <pl-figure file-name="false.png" inline="true"></pl-figure></pl-answer>
+        </pl-multiple-choice>
+        <br>
+        <p>
+            Use <code>inline="true"</code> to embed a figure inline with the rest of the elements, like texts.
+            For example, the "true" image is displayed with:
+        </p>
+        <pl-code language="html">&lt;pl-figure file-name="true.png" inline="true"&gt;&lt;/pl-figure&gt;</pl-code>
```
Comment (nicknytko) [inline exampleCourse/questions/element/figure/question.html:81]:
```suggestion
</p>
```

File: exampleCourse/questions/element/figure/question.html [hunk 3]
```diff
@@ -0,0 +1,83 @@
+<pl-question-panel>
+    <p>
+        Questions below are to showcase different features of <code>pl-figure</code>.
```
Comment (nicknytko) [inline exampleCourse/questions/element/figure/question.html:3]:
```suggestion
        The examples below showcase different features of <code>pl-figure</code>.
```

File: exampleCourse/questions/element/figure/info.json [hunk 4]
```diff
@@ -0,0 +1,7 @@
+{
+    "uuid": "73f032f9-19de-41a6-ae9d-148d1c5264a3",
+    "title": "Element pl-figure: Display a statically or dynamically generated image",
```
Comment (nicknytko) [inline exampleCourse/questions/element/figure/info.json:3]: Can we add this question to the HW3 assessment (PrairieLearn element example gallery)?

Comment (bojinyao) [reply exampleCourse/questions/element/figure/info.json:3]: Added 👍

**Pull request merged**